flask_app/tasks.py

```diff
@@ -121,23 +119,35 @@ def _get_os_type(ssh_client: SSHClient) -> str:
     return stdout.read().decode("utf-8").strip().lower()
 
 
-def _generate_random_combadge_name(stringLength: int) -> str:
-    random_string = ''.join(random.choice(string.ascii_lowercase) for i in range(stringLength))
+def _generate_random_combadge_name(string_length: int) -> str:
+    random_string = ''.join(random.choice(string.ascii_lowercase) for i in range(string_length))
```

Comment:
Frankly speaking, why don't we simply use `uuid` package here, instead of all this mess with random string generation?